Dart/thirdPartySrc/analysisServer/com/google/dart/server/AnalysisServerListener.java

```diff
@@ -222,4 +222,6 @@ public void computedImplemented(String file, List<ImplementedClass> implementedC
    * @param existingImports a map from imported library uri onto the names declared by the imported library.
    */
   void computedExistingImports(String file, Map<String, Map<String, Set<String>>> existingImports);
+
+  public void lspTextDocumentContentDidChange(String url);
```

Comment:
I'll add a `default` empty implementation to this method right to this interface. Otherwise, we'd get Flutter plugin compatibility issues because its `FlutterAnalysisServerListener implements AnalysisServerListener`